packages/track-event-mixin/track-event-gesture.ts

```diff
@@ -0,0 +1,308 @@
+// Check for touch-only devices
+const IS_TOUCH_ONLY = navigator.userAgent.match(/iP(?:[oa]d|hone)|Android/);
+// Number of last N track positions to keep
+const TRACK_LENGTH = 2;
+// Radius for track
+const TRACK_DISTANCE = 5;
```

Comment:
Should these be configurable/customizable? Just a thought.

Reply:
They are not configurable in Polymer Gestures, so I assume they shouldn't.